src/code/TextTableParser.cs

```diff
@@ -0,0 +1,453 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Management.Automation;
+using System.Text.Json;
+
+namespace TextTableParser
+{
+    public class ColumnInfo {
+        public int Start;
+        public int Length;
+        public int SpaceLength;
+    }
+
+    public class HeaderField {
+        public string Name;
+        public int Start;
+        public int Length;
+    }
+
+    public class ObjectPropertyInfo {
+        public HeaderField[] HeaderFields;
+        ObjectPropertyInfo(string line, ColumnInfo[] cInfos)
+        {
+            HeaderFields = new HeaderField[cInfos.Length];
+            for(int i = 0; i < cInfos.Length; i++)
+            {
+                HeaderFields[i] = new HeaderField(){
+                    Name = line.Substring(cInfos[i].Start, cInfos[i].Length).Trim(),
+                    Start = cInfos[i].Start,
+                    Length = cInfos[i].Length
+                };
+
+            }
+        }
+    }
+
+    [Cmdlet("Convert","TextTable")]

```

Comment:
```suggestion
    [Cmdlet("ConvertFrom","TextTable")]
```

During the community call it was suggested to change verb to `ConvertFrom`.